docs/docsite/rst/playbook_guide/playbooks_variables.rst

```diff
@@ -101,17 +101,39 @@ Boolean variables
 =================
 
 Ansible accepts a broad range of values for boolean variables: ``true/false``, ``1/0``, ``yes/no``, ``True/False`` and so on. The matching of valid strings is case insensitive.
-While documentation examples focus on ``true/false`` to be compatible with ``ansible-lint`` default settings, you can use any of the following:
+While documentation examples focus on ``true/false`` to be compatible with ``ansible-lint`` default settings. You can use any of the following values:
+
+Native Boolean
+--------------
+
+Values natively recognized as boolean by Ansible
+
+.. table::
+   :class: documentation-table
+
+   =============================================================================================== ====================================================================
+    Valid values                                                                                    Description
+   =============================================================================================== ====================================================================
+    ``True`` , ``TRUE`` , ``yes`` , ``true`` , ``True`` , ``on``                                     Truthy values
+
+    ``False`` , ``false`` , ``FALSE`` , ``no``, ``off``                                              Falsy values
+
+   =============================================================================================== ====================================================================
+
+Interpretable as Boolean
+------------------------
+
+We need to use the filter ``| bool`` to interpret these strings values as native booleans in logical expressions. Boolean expressions will also work with these values without filter, but interpret them in python style.
 
 .. table::
    :class: documentation-table
 
    =============================================================================================== ====================================================================
     Valid values                                                                                    Description
    =============================================================================================== ====================================================================
-    ``True`` , ``'true'`` , ``'t'`` , ``'yes'`` , ``'y'`` , ``'on'`` , ``'1'`` , ``1`` , ``1.0``     Truthy values
+    ``'true'`` , ``'t'`` , ``'yes'`` , ``'y'`` , ``'on'`` , ``'1'`` , ``1`` , ``1.0``                Truthy values
 
-    ``False`` , ``'false'`` , ``'f'`` , ``'no'`` , ``'n'`` , ``'off'`` , ``'0'`` , ``0`` , ``0.0``   Falsy values
+    ``'false'`` , ``'f'`` , ``'no'`` , ``'n'`` , ``'off'`` , ``'0'`` , ``0`` , ``0.0``               Falsy values
```

Comment:
``` 
y|Y|yes|Yes|YES|n|N|no|No|NO
|true|True|TRUE|false|False|FALSE
|on|On|ON|off|Off|OFF|0|1
```
1.0 and 0.0 are basically ways of writing 1 and 0 as the 0 in both cases after the . is non significant

Reply:
Added all variants in Native and have removed 0.0\1.0 from interpretable as bool